### PR TITLE
Removing the non-linear case from doc strings

### PR DESCRIPTION
**Pull request by christiankral**
This PR fixes some wrong documentation of the `QuasiStatic.FluxTubes` library, which was introduced by copy + paste errors when creating this library.

### CODE REVIEW

File: Modelica/Magnetic/QuasiStatic/FluxTubes/Shapes/FixedShape/Cuboid.mo [hunk 1]
```diff
@@ -1,6 +1,6 @@
 within Modelica.Magnetic.QuasiStatic.FluxTubes.Shapes.FixedShape;
 model Cuboid
-"Flux tube with rectangular cross-section; fixed shape; linear or non-linear material characteristics"
+"Flux tube with rectangular cross-section; fixed shape; linear material characteristics"
```
Comment (beutlich) [inline Modelica/Magnetic/QuasiStatic/FluxTubes/Shapes/FixedShape/Cuboid.mo]:
```suggestion
"Flux tube with rectangular cross-section of fixed shape and linear material characteristics"
```

File: Modelica/Magnetic/QuasiStatic/FluxTubes/Shapes/FixedShape/GenericFluxTube.mo [hunk 2]
```diff
@@ -1,6 +1,6 @@
 within Modelica.Magnetic.QuasiStatic.FluxTubes.Shapes.FixedShape;
 model GenericFluxTube
-"Flux tube with fixed cross-section and length; linear or non-linear material characteristics"
+"Flux tube with fixed cross-section and length; linear material characteristics"
```
Comment (beutlich) [inline Modelica/Magnetic/QuasiStatic/FluxTubes/Shapes/FixedShape/GenericFluxTube.mo]:
```suggestion
"Flux tube with fixed cross-section, fixed length and linear material characteristics"
```

File: Modelica/Magnetic/QuasiStatic/FluxTubes/Shapes/FixedShape/HollowCylinderAxialFlux.mo [hunk 3]
```diff
@@ -1,6 +1,6 @@
 within Modelica.Magnetic.QuasiStatic.FluxTubes.Shapes.FixedShape;
 model HollowCylinderAxialFlux
-"(Hollow) cylinder with axial flux; fixed shape; linear or non-linear material characteristics"
+"(Hollow) cylinder with axial flux; fixed shape; linear material characteristics"
```
Comment (beutlich) [inline Modelica/Magnetic/QuasiStatic/FluxTubes/Shapes/FixedShape/HollowCylinderAxialFlux.mo]:
```suggestion
"(Hollow) cylinder with axial flux of fixed shape and linear material characteristics"
```

File: Modelica/Magnetic/QuasiStatic/FluxTubes/Shapes/FixedShape/HollowCylinderRadialFlux.mo [hunk 4]
```diff
@@ -1,6 +1,6 @@
 within Modelica.Magnetic.QuasiStatic.FluxTubes.Shapes.FixedShape;
 model HollowCylinderRadialFlux
-"Hollow cylinder with radial flux; fixed shape; linear or non-linear material characteristics"
+"Hollow cylinder with radial flux; fixed shape; linear material characteristics"
```
Comment (beutlich) [inline Modelica/Magnetic/QuasiStatic/FluxTubes/Shapes/FixedShape/HollowCylinderRadialFlux.mo]:
```suggestion
"Hollow cylinder with radial flux of fixed shape and linear material characteristics"
```

**Comment by dietmarw** [comment]
I can approve once @beutlich 's suggestions are accepted.

File: Modelica/Magnetic/QuasiStatic/FluxTubes/Shapes/FixedShape/HollowCylinderRadialFlux.mo (hunk 4 above)
Comment (beutlich) [inline Modelica/Magnetic/QuasiStatic/FluxTubes/Shapes/FixedShape/HollowCylinderRadialFlux.mo]: Not yet resolved.

Comment (beutlich) [inline Modelica/Magnetic/QuasiStatic/FluxTubes/Shapes/FixedShape/HollowCylinderRadialFlux.mo]:
```suggestion
"Hollow cylinder with radial flux of fixed shape and linear material characteristics"
```

Comment (christiankral) [reply Modelica/Magnetic/QuasiStatic/FluxTubes/Shapes/FixedShape/HollowCylinderRadialFlux.mo]: Fix wording

Comment (christiankral) [reply Modelica/Magnetic/QuasiStatic/FluxTubes/Shapes/FixedShape/HollowCylinderRadialFlux.mo]: Fix wording

**Pull request merged**